File: ruby/tutorial/comfy/14-callerid-redirect.rb
# -----------
# redirect
# -----------
call = Tropo::TropoCall.new($incomingCall)

if $currentCall.callerID == '4075551212'
  call.answer
  call.say 'Hello there and goodbye'
  call.hangup
else
  call.redirect 'tel:14075551212'
end

File: python/tutorial/14-callerid-redirect.py
# --------------------------------------------
# redirect a call based on caller ID
# See http://www.tropo.com for more info
# --------------------------------------------

if (currentCall.callerID == '4075551111'):
    answer()
    say("Hello, world!")
    hangup()
else:
    redirect("tel:+14075552222")

